lib/Sitemap/Document/Filter/DocumentTypeFilter.php

```diff
@@ -56,7 +56,7 @@ public function __construct(array $documentTypes = null, array $containerTypes =
 
     public function canBeAdded(AbstractElement $element, GeneratorContextInterface $context): bool
     {
-        if (!$element instanceof Document) {
+        if (!$element instanceof Document || $element instanceof Document\Hardlink\Wrapper\Page) {
```

Comment:
Should we better use `\Pimcore\Model\Document\Hardlink\Wrapper\WrapperInterface` here instead? That would cover all hardlinks, no matter which type, or is there a special reason for `Page` ?

Reply:
you are right, it should cover all hardlinks...changed it.